addons/website_event/views/event_templates.xml

```diff
@@ -626,7 +626,9 @@
 
 <!-- Snippets -->
 <template id="country_events" name="Country Events" inherit_id="website.snippets">
-    <xpath expr="//div[@id='snippet_content']/div[@class='o_panel_body']" position="inside">
+    <xpath expr="//div[@id='snippet_content']/div[@class='o_panel_body']/t[@t-title='Local Events']" position="replace"/>
+
+    <xpath expr="//div[@id='snippet_content']/div[@class='o_panel_body']/t[@t-snippet][last()]" position="after">
```

Comment:
Make this inherit the `website.external_snippets` mentioned in my other comment and simply replace the `t-install`. Also, target the `t-install` with `[@t-install='website_event']` not `[@t-title='...']`.
Same goes for the other snippets of course.